### ISSUE TITLE
Why not install netaddr with ansible local task?

### ISSUE THREAD

**Issue by mihailstoynov** [aalaesar/install_nextcloud#82]
You say this in the README:

```### Python libraries
To use `ipwrap` filter in Ansible, you need to install the netaddr Python library on a computer on which you use Ansible (it is not required on remote hosts). It can usually be installed with either your system package manager or using pip:
```bash
$ pip install netaddr
```

Why not do it with a task automatically? Without netaddr a task on http_ngingx.yml:104 fails

**Comment by aalaesar** [comment]
Hello @mihailstoynov ,
Tank you for this  question.
Sorry for my **VERY** late answer, it's hard to get free time for me since a Year.
Anyway:

**TLDR**: I prefer to leave it to the host's admin


I'm not very confortable with modifying the ansible host automatically.
For many _core_ sys-admin reasons:

- I would like the server owner to make the choice and automate it itself rather than injecting the package, (infra as code etc)
- IMHO It's beyond the scope of an ansible role to affect the Host it's running on, _it should be focused only on its tasks whatever the target host is_.
- many unknowns:
  - the user used to run the playbook may not have privileges to install the package
  - Is it python 2 ? python 3 ? using virtual env ?
  - does the sysadmin use PIP or distribution packages to install python modules ?

I understand that the philosophy of python's "batteries included" is nice; specially for devellopment

But the role is already too much complex to manage this cases

Again, thank you for your interest.

Be safe ,
Regards.
